src/main/java/seedu/address/model/tuition/Tuition.java

```diff
@@ -0,0 +1,126 @@
+package seedu.address.model.tuition;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.address.model.session.Session;
+import seedu.address.model.session.exceptions.DuplicateSessionException;
+import seedu.address.model.student.Address;
+import seedu.address.model.student.Email;
+import seedu.address.model.student.Name;
+import seedu.address.model.student.Phone;
+import seedu.address.model.student.Student;
+
+/**
+ * Represents a {@code Tuition} that contains {@code Student} and a list of {@code Session}.
+ *
+ */
+public class Tuition {
+
+    private final Student student;
+    private List<Session> sessions;
+
+    /**
+     * Constructs a {@code Tuition}.
+     *
+     * Requires that all fields entered to be non null
+     */
+    public Tuition(Student student) {
+        requireAllNonNull(student);
+        this.student = student;
+        this.sessions = new ArrayList<>();
+
+    }
+
+    public Name getName() {
+        return getStudent().getName();
+    }
+
+    public Phone getPhone() {
+        return getStudent().getPhone();
+    }
+
+    public Email getEmail() {
+        return getStudent().getEmail();
+    }
+
+    public Address getAddress() {
+        return getStudent().getAddress();
+    }
+
+    public String getStudyLevel() {
+        return getStudent().getStudyLevel();
+    }
+
+    public Phone getGuardianPhone() {
+        return getStudent().getGuardianPhone();
+    }
+
+    public String getRelationship() {
+        return getStudent().getRelationship();
+    }
```

Comment:
I was thinking of `tuition.getStudent().getName()` but I guess this works well. This is just a design consideration but you can check out "Law of Demeter".